data/complete.queryMap.json

```diff
@@ -171,5 +171,11 @@
+  "deba05ff9418dc7b1561697e35596f50": "query AutosuggestResultsQuery(\n  $query: String!\n  $count: Int!\n) {\n  ...AutosuggestResults_results_1bcUq5\n}\n\nfragment AutosuggestResults_results_1bcUq5 on Query {\n  results: searchConnection(query: $query, mode: AUTOSUGGEST, first: $count, entities: [ARTIST, ARTWORK, FAIR, GENE, SALE, GALLERY, PROFILE]) {\n    edges {\n      node {\n        imageUrl\n        href\n        displayLabel\n        ... on SearchableItem {\n          displayType\n          slug\n        }\n        ... on Node {\n          id\n        }\n        __typename\n      }\n      cursor\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n",
+  "a0e9c6c3c27e5740ad3ac69848382ace": "query AutosuggestResultsPaginationQuery(\n  $query: String!\n  $count: Int!\n  $cursor: String\n) {\n  ...AutosuggestResults_results_1jWD3d\n}\n\nfragment AutosuggestResults_results_1jWD3d on Query {\n  results: searchConnection(query: $query, mode: AUTOSUGGEST, first: $count, after: $cursor, entities: [ARTIST, ARTWORK, FAIR, GENE, SALE, GALLERY, PROFILE]) {\n    edges {\n      node {\n        imageUrl\n        href\n        displayLabel\n        ... on SearchableItem {\n          displayType\n          slug\n        }\n        ... on Node {\n          id\n        }\n        __typename\n      }\n      cursor\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n",
+  "3f3a581209373c22b1945f702e2d513d": "query AutosuggestResultsQuery(\n  $query: String!\n  $count: Int!\n) {\n  ...AutosuggestResults_results_1bcUq5\n}\n\nfragment AutosuggestResults_results_1bcUq5 on Query {\n  results: searchConnection(query: $query, mode: AUTOSUGGEST, first: $count, entities: [ARTIST, ARTWORK, FAIR, GENE, SALE, PROFILE]) {\n    edges {\n      node {\n        imageUrl\n        href\n        displayLabel\n        ... on SearchableItem {\n          displayType\n          slug\n        }\n        ... on Node {\n          id\n        }\n        __typename\n      }\n      cursor\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n",
+  "e4098d5f3df8c301e3129df348d14457": "query AutosuggestResultsPaginationQuery(\n  $query: String!\n  $count: Int!\n  $cursor: String\n) {\n  ...AutosuggestResults_results_1jWD3d\n}\n\nfragment AutosuggestResults_results_1jWD3d on Query {\n  results: searchConnection(query: $query, mode: AUTOSUGGEST, first: $count, after: $cursor, entities: [ARTIST, ARTWORK, FAIR, GENE, SALE, PROFILE]) {\n    edges {\n      node {\n        imageUrl\n        href\n        displayLabel\n        ... on SearchableItem {\n          displayType\n          slug\n        }\n        ... on Node {\n          id\n        }\n        __typename\n      }\n      cursor\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n"
```

Comment:
I'm surprised about all these additions, hmm. I'll take a look and report back.

Reply:
this file is a minor source of churn and confusion. I'm going to add a ticket to configure relay to make sure it is always canonical.

Reply:
https://artsyproduct.atlassian.net/browse/MX-183